.github/workflows/main.yaml

```diff
@@ -44,11 +44,23 @@ jobs:
     - name: Run Tests
       run: ape test
 
-    - name: Deploy SubscriptionManager Locally
-      run: ape run scripts/deploy_subscription_manager.py --network ::test
-      
-    - name: Deploy Token Locally
-      run: ape run scripts/deploy_nucypher_token.py --network ::test
+    - name: Deploy NuCypher Token
+      run: ape run scripts/deploy_nucypher_token.py --network ethereum:local
+
+    - name: Deploy Simple PRE
+      run: ape run scripts/deploy_simple_pre.py --network ethereum:local
+
+    - name: Deploy Staking Escrow
+      run: ape run scripts/deploy_staking_escrow.py --network ethereum:local
+
+    - name: Deploy Subscription Manager
+      run: ape run scripts/deploy_subscription_manager.py --network ethereum:local
+
```

Comment:
There is also a `Coordinator` deployment script, https://github.com/nucypher/nucypher-contracts/blob/main/scripts/deploy_coordinator.py. Should that be included here?

Also @cygnusv, does the coordinator deployment script, need to be updated because PR #75 modified the ordering of parameters for the Coordinator contract constructor?

Reply:
I left out `Coordinator`, `StakeInfo` and some of the other newer ones.

The `Coordinator` deployment script definitely needs to be updated, i think it still references brownie.

Reply:
done

Reply:
@theref not to give you more work, but the README needs to be updated to no longer mention `brownie`. That can be done in this PR or a separate PR. Up to you.